audit_lambda.py

```diff
@@ -147,7 +147,7 @@ def get_github_activity_prs_audit(org, today):
 
 
 def get_dependabot_status(org, today):
-    updated = False
+
```

Comment:
run black plx

Reply:
We have this awesome new thing we are trialling called Git pre-commit hooks, they have a black + flake8 hook 👍 :trollface:

Reply:
Done!